cli/kaos_cli/facades/tests/test_backend_facade.py

```diff
@@ -0,0 +1,368 @@
+from parameterized import parameterized
+import re
+from unittest import TestCase
+
+from kaos_cli.facades.backend_facade import BackendFacade
+from kaos_cli.services.terraform_service import TerraformService
+from kaos_cli.services.state_service import StateService
+from kaos_cli.services.terraform_service import Command
+from kaos_cli.constants import CONFIG_PATH, ACTIVE, DEFAULT, CONTEXTS
+
+
+def backend_facade_test_inputs():
+    return [
+        (
+            'DOCKER', 'backend', 'url', 'token', 'infrastructure', 'kubeconfig', 'user'
+        ),
+        (
+            'GCP', 'backend', 'url', 'token', 'infrastructure', 'kubeconfig', 'user'
+        ),
+        (
+            'AWS', 'backend', 'url', 'token', 'infrastructure', 'kubeconfig', 'user'
+        ),
+        (
+            'Remote1', 'backend', 'url', 'token', 'infrastructure', 'kubeconfig', 'user'
+        ),
+    ]
```

Comment:
If this is not going to change often I recommend changing it to:

```
common = ('backend', 'url', 'token', 'infrastructure', 'kubeconfig', 'user')

return [
    ('DOCKER',) + common,
    ('GCP',) + common,
    ('AWS',) + common,
    ('Remote1',) + common,
]
```

It's less verbose and helps point out the differences between all of them.

Reply:
The above suggestion has now been included